ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/models/participant-data.model.ts

```diff
@@ -1,6 +1,6 @@
 export class ParticipantData {
-  constructor(public dataId: string, public fieldTypeId: string, public data: {}) {
-    this.dataId = dataId;
+  constructor(public participantDataId: string, public fieldTypeId: string, public data: {}) {
+    this.participantDataId = participantDataId;
     this.fieldTypeId = fieldTypeId;
     this.data = data;
```

Comment:
Lines 3-5 can be safely removed.

Reply:
I will change that in my next PR! want to get these changes into develop now